Makefile

```diff
@@ -199,6 +190,19 @@ integration-test-auth-local: PASSWORD=testpass
 integration-test-auth-local: | kill build
 	@echo '-- Running system integration tests using local user auth...'
 	$(RUN_INTEGRATION_TEST)
+	
+install-test: ## Run install/uninstall test in a container
+	@echo '-- Running global install/uninstall test...'
+	$(DASHBOARD_SERVER) -it --rm \
+		-v `pwd`:/src \
+		--user root \
+		--entrypoint /bin/bash \
+		$(DASHBOARD_IMAGE_NAME) \
+		-c 'cd /tmp && \
+			npm install --quiet /src && \
+			test -f ./node_modules/.bin/jupyter-dashboards-server && \
+			./node_modules/.bin/jupyter-dashboards-server --help && \
+			npm uninstall --quiet jupyter-dashboards-server'
```

Comment:
> 1. Make part of integration-test target?

Not as written. It's just an install test. See 2.

> Add additional integration test (or change this one) to install then run server against KG container and test loading of notebook?

Yes. Was thinking about how to do this. Not sure how. Any ideas?

> 3. npm install --quiet isn't very quiet!